src/doc/documentation-guide.md

```diff
@@ -94,6 +95,68 @@ that concatenates the various `README.md` files we have.
 
 # Stylistic Conventions
 
+## General Considerations
+
+Documentation is most effective when it is precise, concise and uniform.
+
+ - **Preciseness**. Vague descriptions do more harm than good as different
```

Comment:
for me, lines 102 to 131 can be removed.  They add no more meaning than the words "precise", "concise", and "uniform".